src/test/librados_test_stub/TestClassHandler.cc

```diff
@@ -30,17 +30,30 @@ void TestClassHandler::open_class(const std::string& name,
                                   const std::string& path) {
   void *handle = dlopen(path.c_str(), RTLD_NOW);
   if (handle == NULL) {
-    derr << "Failed to load class: " << dlerror() << dendl;
+    std::cerr << "Failed to load class: " << name << " (" << path << "): "
+              << dlerror() << std::endl;
     return;
   }
-  m_class_handles.push_back(handle);
+
+  // clear any existing error
+  dlerror();
```

Comment:
the dlerror string is always reset by the following `dlsym()` call, hence it's not necessary.

Reply:
I am just following the man page [1]

[1] https://linux.die.net/man/3/dlerror